Cargo.toml

```diff
@@ -42,6 +42,8 @@ render = [
   "bevy_internal/bevy_ui",
 ]
 
+spirv_shader_passthrough = ["bevy_internal/spirv_shader_passthrough" ]
```

Comment:
Please add this to the feature docs and note that it is incompatible with wasm.

Reply:
And any other system which doesn't support spirv, i.e dx12 and metal, afaik.

Reply:
I think it is only compatible with Vulkan. Maybe easier to say that.

Reply:
@molikto please add documentation of the feature to `docs/cargo_features.md` and make note that it is only supported for Vulkan, if that is indeed correct.

Reply:
@Shfty documentation is still missing here.